src/dotnet/Core/Blobs/Internal/LocalFolderBlobStorage.cs

```diff
@@ -1,27 +1,33 @@
-using System.Net.Mime;
 using ActualChat.IO;
+using Microsoft.AspNetCore.StaticFiles;
 using Stl.IO;
 
 namespace ActualChat.Blobs.Internal;
 
 internal class LocalFolderBlobStorage : IBlobStorage
 {
-    private readonly FilePath _directory;
+    private IContentTypeProvider? _contentTypeProvider;
+    private FilePath Directory { get; }
```

Comment:
The agreement is:
- Resolved services are always props
- Everything else is usual (i.e. can be a private field, property, etc. -whatever is reasonable).

And here it's reversed: _contentTypeProvider must be a property, Directory must be a field.

Reply:
[discussed] now it's good